src/js/core.js

```diff
@@ -333,15 +333,15 @@
         return !this.options.extensions['imageDragging'];
     }
 
-    function createContentEditable(textarea, doc) {
```

Comment:
Shouldn't this be a problem if people were previously using this method by giving `doc`? I guess javascript will just ignore the second parameter?